cmake/developer_package/ncc_naming_style/ncc_naming_style.cmake

```diff
@@ -121,14 +129,18 @@ function(ov_ncc_naming_style)
     endif()
 
     cmake_parse_arguments(NCC_STYLE "FAIL"
-        "FOR_TARGET;SOURCE_DIRECTORY" "ADDITIONAL_INCLUDE_DIRECTORIES;DEFINITIONS" ${ARGN})
+        "FOR_TARGET;SOURCE_DIRECTORY" "STYLE_FILE;ADDITIONAL_INCLUDE_DIRECTORIES;DEFINITIONS" ${ARGN})
```

Comment:
STYLE_FILE should be in group with `FOR_TARGET;SOURCE_DIRECTORY` as an option with a single argument

Reply:
ok, moved/